static-analysis/assert-is-instance-of-any.php.disabled

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Webmozart\Assert\StaticAnalysis\AssertIsInstanceOfAny;
+
+use Webmozart\Assert\Assert;
+
+class A
+{
+}
+
+class B
+{
+}
+
+/**
+ * @param mixed $value
+ *
+ * @return A|B
+ */
+function consume($value)
+{
+    Assert::isInstanceOfAny($value, [A::class, B::class]);
```

Comment:
Tricky one: I'm not sure if `@param array<class-string<T>> $param` can be inferred to `@param array<class-string<A>|class-string<B>> $param` here